site/gatsby-site/src/components/discover/Controls.js

```diff
@@ -36,6 +37,10 @@ const Controls = () => {
         </div>
 
         <div className="flex">
+          <div className="bootstrap place-self-center">
```

Comment:
I think we should stop using `bootstrap` stuff?

Reply:
Changed to flowbite button